upgrading/rosa-upgrading-sts.adoc

```diff
@@ -11,21 +11,32 @@ toc::[]
 
 To plan an upgrade, review the xref:../rosa_architecture/rosa_policy_service_definition/rosa-life-cycle.adoc#rosa-life-cycle[{product-title} update life cycle]. The life cycle page includes release definitions, support and upgrade requirements, installation policy information and life cycle dates.
 
-Upgrades are manually initiated or automatically scheduled. Red Hat Site Reliability Engineers (SREs) monitor upgrade progress and remedy any issues encountered.
+OpenShift update channels are the mechanism that you can use to decide which Red Hat OpenShift Container Platform minor version to update your clusters to. {product-title} only supports updates through the `stable` channel. To learn more about OpenShift update channels and releases, see link:https://docs.openshift.com/container-platform/4.15/updating/understanding_updates/understanding-update-channels-release.html[Understanding update channels and releases].
+
+Red Hat hosts a public OpenShift Update Service (OSUS), which provides a link:https://access.redhat.com/labs/ocpupgradegraph/update_channel[graph] of update possibilities based on the {product-title}(ROSA) release images in the official registry.
+[NOTE]
+====
+Certain upgrade paths available in Red Hat OpenShift Container Platform might not be available in ROSA.
+====
+
+include::modules/rosa-how-upgrades-work.adoc[leveloffset=+1]
 
 [id="rosa-sts-upgrading-a-cluster-with-sts"]
 == Upgrading a ROSA Classic cluster
 
-There are two methods to upgrade Classic {product-title} (ROSA) clusters:
-
-* Individual upgrades through the ROSA CLI (`rosa`)
-* Individual upgrades through the {cluster-manager} console
+You must update {product-title} (ROSA) Classic clusters using the ROSA CLI (`rosa`) or the {cluster-manager} console.
 
 [NOTE]
 ====
-When you follow a scheduled upgrade policy, a delay of an hour or more before the upgrade process begins might occur, even if the upgrade is configured to begin immediately. Additionally, the duration of the upgrade might vary based on your workload configuration.
+The actual start time of the cluster upgrade will be within one hour of the upgrade schedule time. Additionally, the duration of the upgrade might vary based on your workload configuration.
 ====
 
+When a ROSA Classic cluster that uses AWS Security Token Services (STS) is upgraded, the ROSA CLI verifies the account and operator role policies for the chosen cluster are compatible with the target version of the upgrade. If the policies are compatible, the CLI automatically upgrades the IAM policies. If the policies are not compatible with the chosen upgrade version, the CLI prompts you to upgrade them. When scheduling the upgrade, you provide administrative acknowledgment to automatically upgrade these policies, if required.
```

Comment:
the sentence "If the policies are compatible, the CLI automatically upgrades the cluster"

Reply:
"When scheduling the upgrade, you provide administrative acknowledgment to confirm you have reviewed the changes involved with the upgrade, if required."